woocommerce/woocommerce
# revert #39803 Add additional Perf Test Daily run

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Reverts https://github.com/woocommerce/woocommerce/pull/39803



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-daily.yml
2. Under "Run workflow", select this branch (instead of trunk)
3. Run the workflow or [check this existing run](https://github.com/woocommerce/woocommerce/actions/runs/5902162083)
4. Check everything passed ok



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
